data/fields/address.json

```diff
@@ -1,6 +1,6 @@
 {
     "type": "address",
-    "key": "addr",
+    "key": "addr:*",
```

Comment:
<img width="200" alt="image" src="https://github.com/user-attachments/assets/bf2cd939-3265-46dc-a9ef-e279bcef2f3f" />

while this appears to work, it will break [other logic that uses `field.key`](https://github.com/openstreetmap/iD/blob/4a1f27b1f6d6a322bd4563ac41aa7f8beb8edd07/modules/ui/fields/address.js#L113). I think it would be safer to use [the dedicated `reference` attribute](https://github.com/ideditor/schema-builder#reference):

```suggestion
    "key": "addr",
    "reference": { "key": "addr:*" },
```